spec/models/player_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Player, type: :model do
+  context 'Saving Test' do
+    it 'ensures player model save' do
+      player = Player.new(user_id: 1, id_user_type: 1, country_residence: 2, state_residence: 1, city_residence: 3, nationality: 'MEX', state_birth: 12, city_birth: 2,persona_description: 'bla bla', weight: 80, height: 70, preferred_position: 'delantero',telephone: '1234567', achievements: 'testing').save
+      expect(player).to eq(true)
+    end
+  end
+
+  context 'model is updating' do 
+    it 'ensures that model is updating elements' do
+      player = Player.find_by(id:8).update(telephone: '123345')
+      expect(player).to eq(true)
```

Comment:
This test should check if the new phone number is actually recorded for that player.